src/routes/costModels/costModelsDetails/utils/table.tsx

```diff
@@ -48,7 +48,7 @@ export function getRowsByStateName(stateName: string, data: any) {
         item.description,
         item.source_type,
         item.sources.length.toString(),
-        relativeTime(item.updated_timestamp),
+        formatDistanceToNow(new Date(item.updated_timestamp), { addSuffix: true }),
```

Comment:
There are a couple issues with the `formatDistanceToNow` function

1. We were showing the last change in days (91 days ago), now it's shown in months (3 months ago)
2. We need to localize the string

Didn't realize this string wasn't localized before, but it appears that `formatDistanceToNow` accepts a locale option. You can get the current locale, using:

`import { getLocale } from 'components/i18n';`

Reply:
I wonder if we'll need a custom function to use days vs months if we want to keep that format, it doesn't look like it is supported with this package.

Reply:
might be able to do this i guess https://date-fns.org/v2.29.3/docs/intlFormatDistance

it has a `unit` option.. so it could always be days?